tcs/csp_solver_two_tank_tes.cpp

```diff
@@ -317,12 +439,15 @@ double C_storage_tank::get_m_m_calc() //ARD new getter for current mass
 
 double C_storage_tank::m_dot_available(double f_unavail, double timestep)
 {
-	double rho = mc_htf.dens(m_T_prev, 1.0);		//[kg/m^3]
-	double V = m_m_prev / rho;						//[m^3] Volume available in tank (one temperature)
-	double V_avail = fmax(V - m_V_inactive, 0.0);	//[m^3] Volume that is active - need to maintain minimum height (corresponding m_V_inactive)
+	//double rho = mc_htf.dens(m_T_prev, 1.0);		//[kg/m^3]
+	//double V = m_m_prev / rho;						//[m^3] Volume available in tank (one temperature)
+	//double V_avail = fmax(V - m_V_inactive, 0.0);	//[m^3] Volume that is active - need to maintain minimum height (corresponding m_V_inactive)
+
+	double mass_avail = std::max(m_m_prev - m_mass_inactive, 0.0);		//[kg]
+	double m_dot_avail = std::max(mass_avail - m_mass_active * f_unavail, 0.0) / timestep;	//[kg/s]
 
 	// "Unavailable" fraction now applied to one temperature tank volume, not total tank volume
-	double m_dot_avail = fmax(V_avail - m_V_active*f_unavail, 0.0)*rho / timestep;		//[kg/s] Max mass flow rate available
+	//double m_dot_avail = fmax(V_avail - m_V_active*f_unavail, 0.0)*rho / timestep;		//[kg/s] Max mass flow rate available
```

Comment:
remove

Reply:
Prefer to keep old code around until people have exercised the new controller for a while